Project-I-Social-Card/social-card/src/App.css

```diff
@@ -2,3 +2,11 @@
   Write your styles/CSS for App.js here and import into the component that you need 
   IF you'd like to attempt to use less you'll have to research how to do this with Create React App
 */
+.App {
+  
+  border: solid red;
```

Comment:
adding the borders is a great development technique to ensure that your css is working!